### PR TITLE
UIEH-905 No providers/packages/titles found does not show the search term entered.

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
[UIEH-905](https://issues.folio.org/browse/UIEH-905)  No providers/packages/titles found does not show the search term entered.
## Purpose
No Provider/Package/Title results should display the following message: [ No providers/packages/titles found for "<<show the search term entered in search box>>." Please check your spelling and filters. ]

## Screenshots
image

## Tests
image

### CODE REVIEW

File: src/components/no-results-message/noResultsMessage.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import css from './noResultsMessage.css';
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const NoResultsMessage = ({ children, ...rest }) => {
+  return (
+    <div className={css.noResultsMessage} {...rest}>
+      <div className={css.noResultsMessageLabelWrap}>
```
Comment (Vladyslav-Velytskyi) [inline src/components/no-results-message/noResultsMessage.js:15]: @BogdanDenis, @MaksymDryha Should we replace this tag with `<p>` like in `SearchAndSort` component?

File: src/components/no-results-message/noResultsMessage.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@import '@folio/stripes-components/lib/variables';
+
+.noResultsMessage {
+  padding: 15px;
+  color: var(--color-text-p2);
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```
Comment (Vladyslav-Velytskyi) [inline src/components/no-results-message/noResultsMessage.css:10]: `top`, `bottom`, `left`, `right`, properties are used for positioning, not for size of element. In my opinion is better to use `height` and `width` for it

```suggestion
  top: 0;
  width: 100%;
  height: 100%;
```

File: src/components/no-results-message/noResultsMessage.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.noResultsMessageLabelWrap {
+  font-size: var(--font-size-large);
+}
+
+.noResultsMessageIcon,
+.noResultsMessageLabel {
+  margin: 0 4px;
+}
```
Comment (Vladyslav-Velytskyi) [inline src/components/no-results-message/noResultsMessage.css:29]: Just suggestion, if you need space between two elements, better to rewrite it as in example below
```suggestion
.noResultsMessageLabel {
  margin-left: 8px;
}
```
Pros of this variant that we don't need for special class for the icon

File: src/components/no-results-message/noResultsMessage.js (hunk 1 above)
Comment (MaksymDryha) [reply src/components/no-results-message/noResultsMessage.js:15]: @Vladyslav-Velytskyi , makes sense

File: src/components/no-results-message/noResultsMessage.css (hunk 2 above)
Comment (MaksymDryha) [reply src/components/no-results-message/noResultsMessage.css:10]: @Vladyslav-Velytskyi, considering all this stuff with dynamic list height calculations + absolute positioning, I'm not sure if that will work, but we can try.

**Pull request merged**